app/src/main/java/com/nitramite/paketinseuranta/DatabaseHelper.java

```diff
@@ -64,7 +57,7 @@
 
 
     // DATABASE NAME
-    private static final String DATABASE_NAME = "PARCELS.db";
+    public static final String DATABASE_NAME = "PARCELS.db";
```

Comment:
Move this into `Constants.java` instead?

Reply:
Yes not bad idea

Reply:
https://github.com/norkator/paketin-seuranta/pull/35/commits/027415733094de84690c8c022c70e2213e0fce62